internal/driver/driver.go

```diff
@@ -122,6 +133,25 @@ func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
 		d.lc.Errorf("failed to register secret update callback: %v", err)
 	}
 
+	// check to see if rtsp-simple-server file/binary exists
+	_, err = os.Stat(RtspServerCmd)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("%s file cannot be found: %s", RtspServerCmd, err.Error())
+		} else if os.IsPermission(err) {
+			return fmt.Errorf("permission denied for %s: %s", RtspServerCmd, err.Error())
+		} else {
+			return fmt.Errorf("unknown error occurred while checking for rtsp-server binary file %s: %s", RtspServerCmd, err.Error())
+		}
+	}
+
+	rtspProc := exec.Command(RtspServerCmd)
+	rtspProc.Stdout = os.Stdout
```

Comment:
How about standard errors?

Reply:
good point, I will look into having the stderr printed out as well in my next PR

Reply:
Being addressed in https://github.com/edgexfoundry/device-usb-camera/pull/270